handbook/openid-connect/openid-connect.md

```diff
@@ -183,14 +243,23 @@ Define AUTH_OIDC_REDIRECT_URI /login/openidc-redirect
 
 # Whether OIDC is enabled.
 Define AUTH_OIDC_ENABLED
+AUTH_OIDC_ENABLE_CLAIM_INFO
```

Comment:
This line isn't needed. It's the Define below that's correct.
```suggestion

```

Reply:
Fixed!